include/IECoreScene/SceneInterface.h

```diff
@@ -230,9 +233,13 @@ class IECORESCENE_API SceneInterface : public IECore::RunTimeTyped
 		/// Convenience method to determine if a piece of geometry exists without reading it
 		virtual bool hasObject() const = 0;
 		/// Reads the object stored at this path in the scene at the given time.
-		virtual IECore::ConstObjectPtr readObject( double time ) const = 0;
+		/// If provided, the Canceller will periodically be checked, terminating the read with an exception if
+		/// the result is no longer needed.
+		virtual IECore::ConstObjectPtr readObject( double time, const IECore::Canceller *canceller = nullptr ) const = 0;
```

Comment:
I didn't catch this before merging, but this is a breaking change, and is preventing IECoreMaya & IECoreHoudini from compiling